src/internal.c

```diff
@@ -29286,6 +29299,13 @@ static int DoSessionTicket(WOLFSSL* ssl, const byte* input, word32* inOutIdx,
                         ssl->arrays->preMasterSz = private_key->dp->size;
 
                         ssl->peerEccKeyPresent = 1;
+                        
+                    #if defined(WOLFSSL_TLS13) || defined(HAVE_FFDHE)
+                        /* client_hello may have sent FFEDH2048, which sets namedGroup, 
```

Comment:
Should we be testing whether TLS 1.3 is actually being used here vs compiled in?

Reply:
This applies to TLS13 or HAVE_FFDHE, so no check for TLS is required.